### DISCUSSION TITLE
[Feature request] renderPageContent 中 添加一个新的hook beforeRenderPageContent

### DISCUSSION THREAD

**Discussion by zenonux** [vuepress/core#1482] [Q&A] [answered]
### Clear and concise description of the problem

希望在读取md文件后，对md文件本身做一些处理之后，再丢给markdown-it去解析

### Suggested solution

```ts
function beforeRenderPageContent(content: string) {
  return content;
}
```

**Comment by Mister-Hope** [comment, marked as answer]
一个markdown-it插件可以做到，可以参考 https://mdit-plugins.github.io/include.html 作为示例。

你可以在 normalize 前后添加 CoreRule 直接修改 markdown 文件内容。

**Reply by zenonux** [reply]
确实可以，实现了我想要的功能，感谢
